pkg/health/clusterStatsHandler.go

```diff
@@ -94,10 +94,16 @@ func ProcessClusterStatsHandler(ctx *fasthttp.RequestCtx, myid uint64) {
 	httpResp.QueryStats["Query Count Since Restart"] = queryCount
 	httpResp.QueryStats["Query Count Since Install"] = queriesSinceInstall
 
+	if queriesSinceInstall > 1 {
+		httpResp.QueryStats["Average Query Latency (since install)"] = fmt.Sprintf("%v", utils.ToFixed(totalResponseTimeSinceInstall/float64(queriesSinceInstall), 3)) + " ms"
+	} else {
+		httpResp.QueryStats["Average Query Latency (since install)"] = fmt.Sprintf("%v", utils.ToFixed(totalResponseTimeSinceInstall, 3)) + " ms"
+	}
+
 	if queryCount > 1 {
-		httpResp.QueryStats["Average Query Latency (since install)"] = fmt.Sprintf("%v", utils.ToFixed(totalResponseTime/float64(queryCount), 3)) + " ms"
+		httpResp.QueryStats["Average Query Latency (since restart)"] = fmt.Sprintf("%v", utils.ToFixed(totalResponseTimeSinceRestart/float64(queryCount), 3)) + " ms"
 	} else {
-		httpResp.QueryStats["Average Query Latency (since install)"] = fmt.Sprintf("%v", utils.ToFixed(totalResponseTime, 3)) + " ms"
+		httpResp.QueryStats["Average Query Latency (since restart)"] = fmt.Sprintf("%v", utils.ToFixed(totalResponseTimeSinceRestart, 3)) + " ms"
 	}
```

Comment:
On lines 97 and 103, why is the condition `> 1` instead of `> 0`?

Reply:
We are using `>1` instead of `>0` to ensure that the average latency calculation is only performed when there are multiple queries and dividing by 1 is redundant because the result would be the same as the original value.